snowpack/src/config.ts

```diff
@@ -256,19 +255,26 @@ function loadPlugins(
     return plugin;
   }
 
-  function loadPluginFromConfig(name: string, options?: any): SnowpackPlugin {
-    const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
+  function loadPluginFromConfig(pluginLoc: string, options?: any): SnowpackPlugin {
+    if (!path.isAbsolute(pluginLoc)) {
```

Comment:
This should always be the case, but if this weren't true `require.resolve()` may hide the fact that this was happening with undefined behavior / wrong plugin versions. Better to be aware of the bug/regression in development if it ever happens.